src/main/resources/default/templates/biz/process/process-details.html.pasta

```diff
@@ -24,13 +24,16 @@
         <i:if test="!files.isEmpty()">
             <w:navbox labelKey="Process.files">
                 <i:for type="sirius.biz.storage.layer2.Blob" var="file" items="files">
-                    <li>
-                        <a href="@file.url().buildURL()" class="link" target="_blank">
-                            <i class="fa fa-file"></i>
-                            @file.getFilename()
-                            <small class="muted">(@formatSize(file.getSize()))</small>
-                        </a>
-                    </li>
+                    <i:local name="url" value="file.url().asDownload().buildURL().orElse(null)"/>
```

Comment:
Why not properly use the optional here by checking `isPresent` in the if instead of forcing it to null?